app/models/role.rb

```diff
@@ -8,4 +8,10 @@ class Role < ApplicationRecord
   has_many :users
 
   scope :admin, -> { find_by(name: ADMIN) }
+
+  def self.create_default_roles(account)
+    Role::ALL_ROLES.each do |role|
```

Comment:
We can remove "Role::" if we are accessing in same class

Reply:
Done.